PWG/DevNanoAOD/AliNanoAODReplicator.h

```diff
@@ -99,13 +101,17 @@ class AliNanoAODReplicator : public AliAODBranchReplicator
   Bool_t IsParticleSelected(Int_t i);
   void CreateLabelMap(const AliAODEvent& source);
   Int_t GetNewLabel(Int_t i);
+  void RelabelAODPhotonCandidates(AliAODConversionPhoton *PhotonCandidate);
   void FilterMC(const AliAODEvent& source);
   AliAODVertex* CloneAndStoreVertex(AliAODVertex* toClone);
  
   AliAnalysisCuts* fTrackCuts; // decides which tracks to keep
   AliAnalysisCuts* fV0Cuts;    // decides which V0s to keep
   AliAnalysisCuts* fCascadeCuts; // decides which cascades to keep
   AliAnalysisCuts* fConversionPhotonCuts; // decides which conversion photons to keep
+  AliAnalysisNanoAODMCParticleCuts* fMCParticleCuts;  // decides which particles from the MC stack to keep
+                                                      // Here we need the actual object because we retrieve the MC
+                                                      // matching of the V0s from here
   
```

Comment:
Same here

Reply:
I know, but please read the comment ;)
Same for your comment above

Reply:
ok, wenn ich das in der cxx caste funktioniert das auch mit nem AliAnalysisCuts object! cool